engine/api/scheduler/executer.go

```diff
@@ -76,6 +76,9 @@ func ExecuterRun(DBFunc func() *gorp.DbMap, store cache.Store) ([]sdk.PipelineSc
 			log.Warning("ExecuterRun> %s", err)
 			return nil, err
 		}
+
+		// wait, this avoid to re-run this job in another instance of API
+		time.Sleep(1 * time.Second)
```

Comment:
If for 1 loop, their is more than 60 pipelines to run; it will take 1 minutes to process all of them; and the ticker tick every 10 second, so it will burn :fire: